### PR TITLE
Set authenticated user's email visibility via API

### PR DESCRIPTION
**Pull request by Sairaviteja27**
Fixes: https://github.com/google/go-github/issues/2688

### CODE REVIEW

**Comment by gmlewis** [comment]
@Sairaviteja27 - please remove the "Draft" status when you want a PR to be reviewed.

**Comment by Sairaviteja27** [comment]
Hi @gmlewis , marked it for review, thanks.

File: github/users_emails.go
Comment (gmlewis) [inline github/users_emails.go:77]: This endpoint only accepts one of the 4 fields of `UserEmail`: `visibility`.
As such, it can be confusing to the user to have to build a `UserEmail` struct for this API call.
Instead, let's please do the following:

```suggestion
// SetEmailVisibility sets the visibility for the primary email address of the authenticated user.
// `visibility` can be "private" or "public".
//
// GitHub API docs: https://docs.github.com/en/rest/users/emails#set-primary-email-visibility-for-the-authenticated-user
func (s *UsersService) SetEmailVisibility(ctx context.Context, visibility string) ([]*UserEmail, *Response, error) {
	updateVisiblilityReq := &UserEmail{Visibility: &visibility}
```

**Review by gmlewis — CHANGES_REQUESTED** [review]
Thank you, @Sairaviteja27 !

File: github/users_emails.go
Comment (Sairaviteja27) [reply github/users_emails.go]: I have made the suggested changes @gmlewis

**Review by gmlewis — APPROVED** [review]
Thank you, @Sairaviteja27 !
LGTM.

Awaiting LGTM+Approval from any other contributor to this repo before merging.

**Review by valbeat — APPROVED** [review]
LGTM!

**Comment by gmlewis** [comment]
Thank you, @valbeat !
Merging.

**Pull request merged**
